### PR TITLE
Removing the Python 3 blockers from gsutil.py

### PR DESCRIPTION
**Pull request by catleeball**
This is to test the `ci-testing` branch to make sure Kokoro sees
that a PR is open and that it runs integration tests on the PR.

### CODE REVIEW

**Comment by Tenzer** [comment]
[This line](https://github.com/GoogleCloudPlatform/gsutil/blob/master/setup.py#L158) would also have to be updated, otherwise you can't install the package on Python 3.

File: gsutil.py [hunk 1]
```diff
@@ -22,8 +22,8 @@
 
 # TODO: gsutil-beta: Distribute a pylint rc file.
 
-if not (2, 6) <= sys.version_info[:3] < (3,):
-  sys.exit('gsutil requires python 2.6 or 2.7.')
+# if not (2, 6) <= sys.version_info[:3] < (3,):
+#   sys.exit('gsutil requires python 2.6 or 2.7.')
```
Comment (walkerjoe) [inline gsutil.py]: We should probably still have some requirements on the python version, like 2.7 or 3.4+.
```
if not ((2, 7) <= sys.version_info[:3] < (3,) or (3, 4) <= sys.version_info[:3] < (4,):
    sys.exit('gsutil requires python 2.7 or 3.4+.')
```

Comment (catleeball) [reply gsutil.py]: Smart thinking! Or even a warning that we only support 2.7, >=3.5 and that unexpected behavior may occur.

**Comment by catleeball** [comment]
Thanks for the feedback @walkerjoe and @Tenzer ! I'll fix these comments ASAP. This branch is mainly to test our CI testing tool which I'm resolving some permissions issues with, so it may be a smidge before these commits are merged into master branch. :smiley:

**Comment by Tenzer** [comment]
@catleeball You're welcome. I came to look into this because I saw that we weren't using the latest version of the library, and was wondering why things started failing when I tried to use the latest version - when I then realised that version 4.35, 4.36 and 4.37 doesn't allow being installed on Python 3 which seemed odd to me.

**Comment by catleeball** [comment]
@Tenzer Definitely! We're currently implementing Python 2 / 3 compatibility; if you're curious you can take a look at our `py-six-current` branch. :)

File: gsutil.py (hunk 1 above)
Comment (houglum) [reply gsutil.py]: I think we should require the Python version to be ==2.7 or >=3.5, since that's all we officially develop for and test.

Comment (catleeball) [reply gsutil.py]: Sounds good to me! I'll keep the blocker when we ship things. I'm not planning to merge this PR to master ever, it's just Kokoro testing,

**Comment by catleeball** [comment]
Closing this, will continue Kokoro testing in possible future PRs that I'll label [TEST - KOKORO - DO NOT MERGE] :smiley:

**Pull request closed without merging**